wearApp/src/main/java/dev/johnoreilly/starwars/wearApp/MainActivity.kt

```diff
@@ -43,49 +51,58 @@ sealed class Screen(val title: String) {
     object Lists : Screen("Lists")
 }
 
-@OptIn(ExperimentalWearMaterialApi::class)
 @Composable
-fun MainLayout() {
-    val navController = rememberSwipeDismissableNavController()
+fun MainLayout(navController: NavHostController) {
     val repo = rememberStarWarsRepository()
 
     val people by repo.people.collectAsState(emptyList())
     val films by repo.films.collectAsState(emptyList())
 
-    val pagerState = rememberPagerState(initialPage = 0)
-    val peopleScrollState = rememberScalingLazyListState()
-    val filmScrollState = rememberScalingLazyListState()
-
-    Scaffold(
-        vignette = { Vignette(vignettePosition = VignettePosition.TopAndBottom) },
-        positionIndicator = {
-            if (pagerState.currentPage == 0) {
-                PositionIndicator(scalingLazyListState = peopleScrollState)
-            } else {
-                PositionIndicator(scalingLazyListState = filmScrollState)
-            }
-        }
-    ) {
-        SwipeDismissableNavHost(navController, startDestination = Screen.Lists.title) {
+    Scaffold(vignette = { Vignette(vignettePosition = VignettePosition.TopAndBottom) }) {
+        SwipeDismissableNavHost(
+            navController = navController,
+            startDestination = Screen.Lists.title
+        ) {
             composable(Screen.Lists.title) {
```

Comment:
I'll follow up on this, probably with a `pager()` navigation DSL that hides flipping between different position indicator states etc.  But will be a follow up.